core/src/main/kotlin/org/evomaster/core/problem/external/service/ExternalServiceHandler.kt

```diff
@@ -0,0 +1,168 @@
+package org.evomaster.core.problem.external.service
+
+import com.github.tomakehurst.wiremock.WireMockServer
+import com.github.tomakehurst.wiremock.core.WireMockConfiguration
+import com.github.tomakehurst.wiremock.extension.responsetemplating.ResponseTemplateTransformer
+import com.google.inject.Inject
+import org.evomaster.core.EMConfig
+import org.evomaster.core.problem.external.service.ExternalServiceUtils.generateRandomIPAddress
+import org.evomaster.core.problem.external.service.ExternalServiceUtils.isAddressAvailable
+import org.evomaster.core.problem.external.service.ExternalServiceUtils.nextIPAddress
+
+import com.github.tomakehurst.wiremock.client.WireMock.*
+import org.evomaster.core.problem.external.service.ExternalServiceUtils.isReservedIP
+import org.evomaster.core.search.service.Randomness
+
+class ExternalServiceHandler {
+    /**
+     * This will hold the information about the external service
+     * calls inside the SUT. Information will be passed to the core
+     * through AdditionalInfoDto and will be captured under
+     * AbstractRestFitness and AbstractRestSample for further use.
+     *
+     * TODO: This is not the final implementation need to refactor but
+     * the concept is working.
+     */
+
+    /**
+     * Contains the information about each external calls made
+     */
+    private val externalServiceInfos: MutableList<ExternalServiceInfo> = mutableListOf()
+
+    private val externalServices: MutableList<ExternalService> = mutableListOf()
+
+    private var lastIPAddress : String = ""
+
+    /**
+     * Contains hostname against to WireMock instance mapping
+     */
+    private val externalServiceMapping: MutableMap<String, String> = mutableMapOf()
+
+    @Inject
+    private lateinit var randomness: Randomness
+
+    @Inject
+    private lateinit var config : EMConfig
+
```

Comment:
put the `@Inject` variables at beginning of list of variables, ie move up before `externalServiceInfos`